client/components/forms/form-country-select/index.jsx

```diff
@@ -34,7 +34,7 @@ export default localize( React.createClass( {
 
 		return (
 			<select
-				{ ...omit( this.props, 'className' ) }
+				{ ...omit( this.props, [ 'className', 'countriesList', 'translate', 'moment', 'numberFormat' ] ) }
```

Comment:
:rofl: Actually, you did change my mind - very good point about the additional props we pass down to the raw HTML element, I've missed that in my review. So, while I think we all agree that it's not the "purest" approach, "blacklisting" is indeed the best approach in this case 😭